doc/dev/debug.md

```diff
@@ -99,8 +99,11 @@ $ gdb --args bin/nvfuser_tests --gtest_filter=<FILTER>
 
 ```shell
 $ python setup.py develop --build-with-asan
+# or if you are on the PJNL docker image, NVFUSER_BUILD_EXTRA_PYTHON_ARGS='--build-with-asan' _bn
+
+$ LD_PRELOAD=$(gcc -print-file-name=libasan.so) ASAN_OPTIONS=protect_shadow_gap=0 <CMD>
+# or LD_PRELOAD=$(clang -print-file-name=libclang_rt.asan-x86_64.so) ASAN_OPTIONS=protect_shadow_gap=0 <CMD>
 # The ASAN_OPTIONS is needed to work around https://github.com/google/sanitizers/issues/629.
-$ ASAN_OPTIONS=protect_shadow_gap=0 <CMD>
```

Comment:
Is this WAR no longer necessary? If so, should we remove the comment above too?

Reply:
That is still needed and appears in the command above. I added LD_PRELOAD to the beginning so you might have missed that

Reply:
You're right.